oslabs-beta/apollo-epoch
# Webpack Complete

This is working and configured based on the TESTSETUP directory. When we have our folder structure in place, we can reconfigure the locations. 

To test: 
-Run npm run build. You'll produce a production build folder.
-Create an extension -- Load unpack pointed at said build folder. 
-Pin the new extension to your toolbar (right click the puzzel icon and you should see your extension with a greyed out pin, click that). 
-Click your extension icon. You should see hello world. 
-Now npm start
-The builder folder should rebuild.
-Click your icon again, you should still see the hello world string.
-Navigate to testSetup/components/App.jsx.
-Change the hello world string.
-Click your icon
-The string should be updated with your change
